products/llm_observability/frontend/LLMObservabilityTraceScene.tsx

```diff
@@ -246,14 +246,29 @@ const TreeNode = React.memo(function TraceNode({
     topLevelTrace,
     item,
     isSelected,
+    aggregation,
 }: {
     topLevelTrace: LLMTrace
     item: LLMTrace | LLMTraceEvent
     isSelected: boolean
+    aggregation?: SpanAggregation
 }): JSX.Element {
-    const totalCost = 'properties' in item ? item.properties.$ai_total_cost_usd : item.totalCost
-    const latency = 'properties' in item ? item.properties.$ai_latency : item.totalLatency
-    const usage = formatLLMUsage(item)
+    const totalCost =
+        aggregation?.totalCost ?? ('properties' in item ? item.properties.$ai_total_cost_usd : item.totalCost)
+    const latency =
+        aggregation?.totalLatency ?? ('properties' in item ? item.properties.$ai_latency : item.totalLatency)
```

Comment:
That's not three potential sources for one value, is there a way to unify this on the interface level somehow? (Off the cuff idea: set `totalCost`/`totalCost` on the plain span `item`s _themselves_, as a Kea selector)